### ISSUE TITLE
Is It possible to custom short-deck card range?

### ISSUE THREAD

**Issue by felix5572** [bupticybee/TexasSolver#85]
Just like playing with cards higher than 7  (instead of higher than 6 )?

**Comment by bupticybee** [comment]
> Just like playing with cards higher than 7 (instead of higher than 6 )?

Yes, for me it's convenient, just some modification of code and I know how to. However I think  it's hard for you to do the job.
